components/brave_vpn/BUILD.gn

```diff
@@ -109,7 +109,10 @@ source_set("unit_tests") {
 
     # Currently, tests are for desktop only (fails on Android).
     if (is_win || is_mac) {
-      sources += [ "brave_vpn_unittest.cc" ]
+      sources += [
+        "brave_vpn_unittest.cc",
+        "skus_utils_unittest.cc",
```

Comment:
I think `skus_utils_unittest.cc` should be included in `//brave/components/skus/browser`?

Reply:
done